### ISSUE TITLE
FE: Move every form in the code base into useForm from antd

### ISSUE THREAD

**Issue by palashgdev** [SigNoz/signoz#2150]
## Is your feature request related to a problem?

*Please describe.*
Move every form in the code base into useForm from antd

**Comment by RubensRafael** [comment]
Hello @palashgdev . This issue looks interesting. What do you think about splitting this issue into smaller ones? Maybe separated by areas where the forms are, with some video showing where they are on the page. It would be really cool to work on that.

**Comment by palashgdev** [comment]
Hi @RubensRafael 
allow me some time I will update you by EOD

**Comment by palashgdev** [comment]
You can start with 

- [ ] Login https://github.com/SigNoz/signoz/tree/develop/frontend/src/pages/Login
- [ ] Signup https://github.com/SigNoz/signoz/blob/develop/frontend/src/pages/SignUp/SignUp.tsx
- [ ] Reset Password https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/ResetPassword/index.tsx

**Comment by palashgdev** [comment]
Thanks for sharing @RubensRafael @gitstart but there are more form that we can transform let cover those into another PR/issue

**Comment by RubensRafael** [comment]
@RubensRafael 
Hello @palashgdev , I found other forms, 3 of them are related to each other, but the other two are not.

https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx
https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/Licenses/ApplyLicenseForm.tsx

All these are in the same folder
https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/RuleOptions.tsx
https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/QuerySection.tsx
https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/BasicInfo.tsx

**Comment by palashgdev** [comment]
> @RubensRafael Hello @palashgdev , I found other forms, 3 of them are related to each other, but the other two are not.
> 
> https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/Licenses/ApplyLicenseForm.tsx
> 
> All these are in the same folder https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/RuleOptions.tsx https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/QuerySection.tsx https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/FormAlertRules/BasicInfo.tsx

yes @RubensRafael you are right we can also transform other 3 also

**Comment by GermaVinsmoke** [comment]
Hi @palashgdev 
Can take this one, if #2658 looks okay.

**Comment by palashgdev** [comment]
sure @GermaVinsmoke

**Comment by IMPranshu** [comment]
I was going through this issue and I think it should be closed because the PR has been merged for this.
